frontend/src/views/LoginForm.js

```diff
@@ -25,49 +63,67 @@ const LoginForm = observer(({ location }) => {
   if (rootStore.authStore.isAuthenticated) return <Redirect to={from} />
 
   return (
-    <Container className="login-form">
-      <Grid
-        container
-        component="form"
-        className="login-form__form"
-        onSubmit={login}
+    <>
+      <AppBar
+        title="Prevention Point"
+        position="fixed"
+        className={classes.appBar}
       >
-        <Grid item xs={12}>
-          <FormControl error={rootStore.authStore.error}>
-            <InputLabel htmlFor="username">Username</InputLabel>
-            <PrevPointInput
-              id="username"
-              name="username"
-              type="username"
-              value={username}
-              onChange={e => setUsername(e.target.value)}
-              required
-            />
-          </FormControl>
-        </Grid>
-        <Grid item xs={12}>
-          <FormControl error={rootStore.authStore.error}>
-            <InputLabel htmlFor="password">Password</InputLabel>
-            <PrevPointInput
-              id="password"
-              name="password"
-              type="password"
-              value={password}
-              onChange={e => setPassword(e.target.value)}
-              required
-            />
-          </FormControl>
-        </Grid>
-        <Grid item xs={12}>
-          <PrevPointButton type="submit">Sign In</PrevPointButton>
-          {rootStore.authStore.error && (
-            <PrevPointCopy className="login-form__error">
-              Incorrect Username or password
-            </PrevPointCopy>
-          )}
+        <Toolbar className={classes.navbarToolbar}>
+          <img
+            src={PreventionPointLogo}
+            alt="Prevention Point Logo"
+            className="navbar__logo"
+            width="150"
+          />
+        </Toolbar>
+      </AppBar>
```

Comment:
The diff looks a little weird, but all I did was add a basic header with just the logo in it. It didn't look like any of the other header elements were functional on the login page, so I didn't think they were necessary. I tried just including the Navbar component, and while the site seemed to function fine this way, I got a test error back saying that withRouter was being used outside of a Route.